.github/workflows/superset-python-integrationtest.yml

```diff
@@ -150,7 +150,9 @@ jobs:
       SUPERSET_CONFIG: tests.integration_tests.superset_test_config
       REDIS_PORT: 16379
       SUPERSET__SQLALCHEMY_DATABASE_URI: |
-        sqlite:///${{ github.workspace }}/.temp/unittest.db
+        sqlite:///${{ github.workspace }}/.temp/superset.db
+      SUPERSET__SQLALCHEMY_EXAMPLES_URI: |
```

Comment:
This threw me for a curveball as for some tests the metadata database _only_ contained example tables. Per [here](https://github.com/apache/superset/blob/9033e72679fad7efdabc0f5b28607d974a7b5e1f/tests/integration_tests/superset_test_config.py#L46-L48), if undefined the metadata and example databases are the same and given that SQLite doesn't support schemas, the metadata database was likely being overwritten by the examples database.